test/test/helpers/use-sinon/index.js

```diff
@@ -66,3 +66,18 @@ export function useSandbox( config, sandboxCallback = noop ) {
 		}
 	} );
 }
+
+/**
+ * Creates a sinon spy out of responseFunction and wraps it in layers of function calls, like wpcom does.
+ * Returned object stubs wpcom.
+ * responseSpy attribute of returned object provides reference to sinon spy.
+ * @param {Array} path
+ * @param {Function} responseFunction
+ * @returns {Object} wpcom-like wrapped object
+ */
+export function createWpcomSpy( path, responseFunction ) {
```

Comment:
Do we need to support the following types of wpcom calls where arguments are provided during the chain?

```js
wpcom.site( 2916284 ).post( 342 ).comments()
```